app/javascript/App/bundles/signalsPage/SignalExamplePanel.tsx

```diff
@@ -72,6 +77,7 @@ interface Props extends DeviceContextType {
   title: string
   buyDescription?: string
   sellDescription?: string
+  active?: boolean
```

Comment:
I personally prefer boolean variables to be prefixed with "is", i.e. `isActive` but I think it's fine here.